client/app/styles/modules/_m-actions-milestones.scss

```diff
@@ -14,18 +14,14 @@
   position: relative;
   box-shadow: inset rem-calc(28) 0 0 $body-background, inset rem-calc(30) 0 0 $medium-gray;
 
-  &.past {}
-
-  &.present {
+  &.green-glow {
```

Comment:
Renaming this effect. It's not good to use CSS for logic, and renaming this variable to reflect the visual style
![image](https://user-images.githubusercontent.com/3311663/107180725-26abd500-698e-11eb-8d68-8d2fcb7591df.png)